.github/workflows/pr.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: pr
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      changed-go: ${{ steps.changed-paths.outputs.go == 'true' }}
+    steps:
+      -
+        uses: actions/checkout@v4
+      -
```

Comment:
nit: any reason the dashes are on their own lines?

Reply:
Yeah, I've been seeing this in some newer actions code. I like this style because it makes moving lines around easier since the dash is its own line